src/QueryDataTable.php

```diff
@@ -48,6 +49,13 @@ class QueryDataTable extends DataTableAbstract
      */
     protected bool $keepSelectBindings = false;
 
+    /**
+     * Flag to ignore the selects in count query.
+     *
+     * @var bool
+     */
+    public bool $ignoreSelectInCountQuery = false;
```

Comment:
Can we make this protected?

```suggestion
    protected bool $ignoreSelectInCountQuery = false;
```